bitnami/redis/templates/networkpolicy.yaml

```diff
@@ -65,15 +61,18 @@ spec:
               {{- end }}
           {{- end }}
         {{- end }}
+        {{- if .Values.networkPolicy.additionalRules }}
```

Comment:
These additional rules are only for the `ingress` section, but you cannot set anything for the `egress` section. In the JupyterHub chart we use `extraIngress` and `extraEgress`. I think that makes more sense.